src/cts/src/TechChar.cpp

```diff
@@ -484,9 +484,33 @@ void TechChar::initCharacterization()
     masterNames_.insert(masterString);
   }
 
-  std::string bufMasterName = masterVector[0];
-  charBuf_ = db_->findMaster(bufMasterName.c_str());
-
+  float maxBuffCap = 0.0;
+  std::string bufMasterName; 
+  for (int i = 0; i < masterVector.size(); i++){
+      float maxCap= 0.0;
+      bool maxCapExist= false;
+      odb::dbMaster* buf = db_->findMaster(masterVector[i].c_str());
+      sta::Cell* masterCell = db_network_->dbToSta(buf);
+      sta::LibertyCell* libertyCell = db_network_->libertyCell(masterCell);
+      sta::LibertyPort *input, *output;
+      libertyCell->bufferPorts(input, output);
+      output->capacitanceLimit(sta::MinMax::max(), maxCap, maxCapExist);
+      if( maxCap >= maxBuffCap){
```

Comment:
You need to have !maxCapExists by using the library default as is done in the code further down.